tests/unittests/test_completers.py

```diff
@@ -350,7 +350,7 @@ def test_timestamp_completer_datetime_format_time_completion():
             text="1581033600000",
             start_position=-10,
             display=FormattedText([("", "1581033600000")]),
-            display_meta="2020-02-07T00:00:00+00:00",
+            display_meta="2020-02-07 00:00:00+00:00",
```

Comment:
The difference came from here, we use `str(pendulum_obj)` as the display: https://github.com/laixintao/iredis/blob/3926c16faeaba43e62c9539a41f7d698e8a35706/iredis/completers.py#L131

```python
import pendulum

dt = pendulum.parse("2023-12-01")
print(dt)


# pendulum 2.x: 2023-12-01T00:00:00+00:00
# pendulum 3.x: 2023-12-01 00:00:00+00:00
```

A think a better fix would be, also change the code on https://github.com/laixintao/iredis/blob/3926c16faeaba43e62c9539a41f7d698e8a35706/iredis/completers.py#L131 to `dt.format('YYYY-MM-DD HH:mm:ss')`, so that we do not depend on the default behavior, what do you think?